testing_dir/minesweeper.py

```diff
@@ -36,7 +36,7 @@ def print_stats():
 def update_player_pos(direction):
     x, y = player_pos
     if direction == "up":
-        y -= 1
+        y -= 2  # Error: should be 1
```

Comment:
It seems the movement for the "up" direction incorrectly reduces `y` by 2. If the intention is to move by 1 unit, adjust it accordingly.
```diff
-        y -= 2  # Error: should be 1
+        y -= 1  # Error: should be 1
```